go/interpreter/lfvm/interpreter.go

```diff
@@ -604,15 +603,13 @@ func steps(c *context, oneStepOnly bool) {
 	}
 }
 
+// satisfiesStackRequirements checks if the current stack size satisfies the
+// stack requirements of the given OpCode.
+// caller should handle false return as an error.
 func satisfiesStackRequirements(c *context, op OpCode) bool {
```

Comment:
Now that the function does not modify the context, we could pass only the stack size as an argument instead of the full mutable context